### PR TITLE
hw5

### PR DESCRIPTION
**Pull request by MaratMukhutdinov**
Implemented Homework5

### CODE REVIEW

File: Homework5/Application.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package Homework5;
+import static Homework5.VendingMachine.addMoney;
+import static Homework5.VendingMachine.menu;
+
+public class Application {
+    public static void main(String[] args) {
+        menu();
+        addMoney();
+
```
Comment (BulatMukhutdinov) [inline Homework5/Application.java]: redundant empty lines

Comment (MaratMukhutdinov) [reply Homework5/Application.java]: Fixed

File: Homework5/Drink.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+public enum Drink {
+    COCACOLA("CocaCola", 60),
+    MOUNTAINDEW("Mountain Dew", 55),
+    FANTA("Fanta", 55),
+    PEPSI("Pepsi", 55),
+    LIPTONTEA("Lipton Tea", 50),
+    MIRINDA("Mirinda", 60);
+    private String description;
+    private int cost;
+
```
Comment (BulatMukhutdinov) [inline Homework5/Drink.java]: empty line

File: Homework5/Drink.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
+package Homework5;
+
+public enum Drink {
+    COCACOLA("CocaCola", 60),
+    MOUNTAINDEW("Mountain Dew", 55),
+    FANTA("Fanta", 55),
+    PEPSI("Pepsi", 55),
+    LIPTONTEA("Lipton Tea", 50),
+    MIRINDA("Mirinda", 60);
+    private String description;
```
Comment (BulatMukhutdinov) [inline Homework5/Drink.java]: better add a new line

File: Homework5/Drink.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    private int cost;
+
+
+    Drink(String description, int cost) {
+        this.description = description;
+        this.cost = cost;
+
```
Comment (BulatMukhutdinov) [inline Homework5/Drink.java]: empty line

File: Homework5/Drink.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    }
+
+    public String getDescription() {
+        return description;
+    }
+    public int getCost() {
+        return cost;
+    }
+
```
Comment (BulatMukhutdinov) [inline Homework5/Drink.java]: empty line

File: Homework5/Drink.java [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+    Drink(String description, int cost) {
+        this.description = description;
+        this.cost = cost;
+
+    }
+
+    public String getDescription() {
+        return description;
+    }
+    public int getCost() {
```
Comment (BulatMukhutdinov) [inline Homework5/Drink.java]: add a new line

File: Homework5/VendingMachine.java [hunk 7]
```diff
@@ -0,0 +1,76 @@
+package Homework5;
+
+import java.util.Scanner;
+
+import static Homework5.Drink.*;
+
+class VendingMachine {
+
+    static String brand;
```
Comment (BulatMukhutdinov) [inline Homework5/VendingMachine.java]: remove all static modifiers from the class and make fields private

File: Homework5/VendingMachine.java [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+            System.out.println(position + ". " + drink.getDescription() + ", цена: " + drink.getCost() + " рублей.");
+            position++;
+        }
+
+        Scanner scan = new Scanner(System.in);
+        drinkChoose = scan.nextInt();
+
+        if (drinkChoose > 6 || drinkChoose < 0) {
+            System.out.println("Вы ввели неккоректные данные!");
+            System.exit(0);
```
Comment (BulatMukhutdinov) [inline Homework5/VendingMachine.java]: don't use system.exit. Prefer exceptions or return some value meaning method failed to complete

Comment (MaratMukhutdinov) [reply Homework5/VendingMachine.java]: While method has been added

File: Homework5/VendingMachine.java (hunk 7 above)
Comment (MaratMukhutdinov) [reply Homework5/VendingMachine.java]: Static fields were removed

File: Homework5/Drink.java (hunk 6 above)
Comment (MaratMukhutdinov) [reply Homework5/Drink.java]: Done

File: Homework5/Drink.java (hunk 5 above)
Comment (MaratMukhutdinov) [reply Homework5/Drink.java]: Deleted

File: Homework5/Drink.java (hunk 4 above)
Comment (MaratMukhutdinov) [reply Homework5/Drink.java]: Deleted

File: Homework5/Drink.java (hunk 3 above)
Comment (MaratMukhutdinov) [reply Homework5/Drink.java]: Added

File: Homework5/Drink.java (hunk 2 above)
Comment (MaratMukhutdinov) [reply Homework5/Drink.java]: Deleted

**Pull request closed without merging**